docker-compose.yml

```
@@ -163,6 +163,24 @@ services:
             - ./docker/haproxy/cors.lua:/etc/haproxy/cors.lua
         scale: -1
 
+    loki:
+      image: grafana/loki:2.4.0
```

Please add to the `.env.example` the appropriate entry for this variable:
```suggestion
      image: ${LOKI_IMAGE:-grafana/loki:2.4.0}
```